LeaguePackets/GamePacketID.cs

```diff
@@ -166,7 +166,7 @@ public enum GamePacketID : uint
         S2C_ToggleInputLockFlag = 0x9B,                 //DONE?
         C2S_SoftReconnect = 0x9C,                       //DONE?
         S2C_CreateTurret = 0x9D,                        //DONE?
-        NPC_Die_Broadcast = 0x9E,                                 //DONE?
+        NPC_Die_Broadcast = 0x9E,                        //DONE?
```

Comment:
There is additional space here.

Reply:
I'll merge this and open an issue about this single space before the `//` comment.

Reply:
Fixed.